activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -40,6 +40,12 @@ def initialize(scope)
       #
       #    User.where.not(name: "Jon", role: "admin")
       #    # SELECT * FROM users WHERE NOT (name == 'Jon' AND role == 'admin')
+      #
+      #    # Notice:
```

Comment:
Thanks for the PR!

I think this could be a little clearer. Can you update this to explain about the behavior of nullable columns rather than just "Notice".

Reply:
Thank you for your advice, and I think you are right. I have tried to make it a little clearer.